src/libs/actions/PersonalDetails.js

```diff
@@ -141,38 +141,6 @@ function setPersonalDetails(details, shouldGrowl) {
         });
 }
 
-/**
- * @param {String} firstName
- * @param {String} lastName
- * @param {String} pronouns
- * @param {Object} timezone
- */
-function updateProfile(firstName, lastName, pronouns, timezone) {
-    API.write('UpdateProfile', {
```

Comment:
I created [this](https://github.com/Expensify/Expensify/issues/247222) GH issue to watch this api command and remove when possible.

Reply:
Hmm this cleanup is supposed to take place in this issue: https://github.com/Expensify/Expensify/issues/231919

You can see my whole tracking list of issues for Account Settings here: https://github.com/Expensify/Expensify/issues/182122